src/types.ts

```diff
@@ -40,9 +40,4 @@ export interface InputConfig {
     minimumSenderBalance: number;
 }
 
-export interface CmdOptions {
-    detached?: boolean;
-    matcher?: RegExp;
-    verbose?: boolean;
-    env?: any;
-}
+
```

Comment:
the `any` in it was causing a lint warning, the I figured that the whole interface was not referenced at all in the code